docs/faq.md

```diff
@@ -10,55 +10,46 @@ hide:
 ## <span class="custom-heading" data-id="1">Why ExpressLRS?</span>
 
 ??? faq "Why ExpressLRS?"
-    ExpressLRS is competing with some of the best commercial solutions out there, but all are expensive 🙁. Well, ELRS is not 😄, it can be set up for around 60$ (Using second-hand r9 hardware). It also is better than many (higher sensitivity, lower latency👌).
-## <span class="custom-heading" data-id="2">Is it better than the commercial solutions?</span>
+    ExpressLRS is a high-performance, open-source radio control protocol built from the ground up to be an extremely fast (**E**xpress), extremely sensitive (**L**ong **R**ange) **S**ystem. ELRS uses Semtech SX12xx LoRa transceivers and a lightweight, highly optimized over-the-air (OTA) protocol to provide superior performance compared to legacy RC links. While ExpressLRS started with DIY and repurposed hardware, a large number of manufacturers are now producing high-quality, low-cost transmitters and receivers to make it easy for anyone to get started!
+## <span class="custom-heading" data-id="2">How does ExpressLRS compare to other systems?</span>
 
-??? faq "Is it better than the commercial solutions?"
-    It is faster than most links out there with a latency of 6.5ms (at 200hz). At 250hz 100mW 2.4GHz it is capable of ranges 30km+.
-## <span class="custom-heading" data-id="3">Which OpenTX version works with ExpressLRS</span>
+??? faq "How does ExpressLRS compare to other systems?"
+    ExpressLRS is capable of extremely high packet rates (up to 1000 Hz) and extreme sensitivity (ranges well over 100+ km have been achieved with 2.4 GHz hardware), making it extremely competitive with any commercially available system. And since ExpressLRS is open-source, it costs much less than other systems too!
+## <span class="custom-heading" data-id="3">Which OpenTX/EdgeTX version do I need and why should I update?</span>
 
-??? faq "Which OpenTX version works with ExpressLRS"
-    [OpenTX 2.3.12](https://www.open-tx.org/downloads.html#Releases23-ref) or newer should work just fine. No need to use OpenTX Nightly, unless you have requirements not present in the Stable versions.
+??? faq "Which OpenTX/EdgeTX version do I need and why should I update?"
+    ExpressLRS requires [OpenTX 2.3.12](https://www.open-tx.org/downloads.html#Releases23-ref) or newer, as it depends on CRSFshot (a.k.a mixer sync) to work properly. However, as OpenTX is no longer being maintained, it is strongly recommended that you update to [EdgeTX](https://github.com/EdgeTX/edgetx) in order to take advantage of the latest performance improvements and bug fixes. 
+	
+	Click [here](https://www.expresslrs.org/quick-start/transmitters/tx-prep.md) to read more about radio firmware requirements and setup.
```

Comment:
should it be relative URL instead of the absolute one?

Reply:
Changed